src/sincosint.jl

```diff
@@ -18,7 +18,7 @@ using Base.Math.@horner
 function sinint(x::Float64)
     t = x*x
     if t ≤ 36.0
-        return x * @horner(t, 1.00000000000000000000E0,
```

Comment:
why change these, there was nothing wrong with just returning the result directly